solanches/tests/test_controller.py

```diff
@@ -1 +1,26 @@
+import pytest
+from unittest import mock
 
+@mock.patch('solanches.models.Comercio')
```

Comment:
Aqui deve ser mockada a função Comercio.get_by_name. Imagino que o retorno dela não seja uma exception, mas sim algo como uma lista vazia ou um dict vazio. Não tenho certeza. Se for uma exception, verifica qual é (do pymongo.erros) e substitui no tratamento do pytest.raises